lib/carin_for_blue_button_test_kit/custom_groups/visual_inspection_and_attestation/v200_server/00_authorization_group/attestation_test_requirement_97.rb

```diff
@@ -16,26 +16,55 @@ class AttestationTestCarinV2Requirement97 < Inferno::Test
 
     verifies_requirements 'hl7.fhir.us.carin-bb_2.0.0@97', 'hl7.fhir.us.carin-bb_2.0.0@102'
 
+    input :carin_server_requirement_97_attestation_options,
+          title: 'Responds with appropriate error codes',
+          description: %(
+            I attest that the Health IT Module returns the following response classes:
+              - (Status 400): invalid parameter
+              - (Status 401/4xx): unauthorized request
+              - (Status 403): insufficient scope
+              - (Status 404): unknown resource
+              - (Status 410): deleted resource
+
+              I further attest that the Health IT Module rejects any unauthorized request by returning an HTTP 401 "Unauthorized",
```

Comment:
There are a few of these "I further attest" in the repo that should probably be updated to the more impersonal "the developer of ..." form as well.

Reply:
Lizzie explained that the difference is intentional:
- "I attest" = visual inspection where the expectation is that the behavior has been observed by the tester.
- "The developer attests" = true attestation without any proof or demonstration.

Lizzie, if the "I attest" stuff is supposed to be visual inspection, should there be more language in there indicating that expectation, like "I further attest that I have observed that the system rejects any unauthorized ..."?